.changelog/4445.yml

```
@@ -0,0 +1,4 @@
+changes:
+- description: replacing playbookId with playbookName in tasks.
```

```suggestion
- description: replacing playbookId with playbookName in tasks.
```
Should be in the template of Fixed an issue where...

---

@maimorag See https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs for all available templates.